tests/kafkatest/tests/connect/connect_test.py

```diff
@@ -171,8 +166,6 @@ def test_skip_and_log_to_dlq(self, error_tolerance, metadata_quorum):
         self.sink = ConnectStandaloneService(self.test_context, self.kafka, [self.OUTPUT_FILE, self.OFFSETS_FILE],
                                              include_filestream_connectors=True)
 
-        if self.zk:
```

Comment:
Do we need this for the cases where ZK remains? (i.e. the Kafka 2.1 case?)

Reply:
See comment above.